.github/workflows/test-c3.yml

```diff
@@ -3,7 +3,7 @@ name: C3 Tests
 on:
   pull_request:
     paths:
-      - packages/create-cloudflare
+      - packages/create-cloudflare/**
 
 env:
   node-version: latest
```

Comment:
This needs to be pinned to a version that is compatible with the engines properties in the package.jsons